frame/assets/src/functions.rs

```diff
@@ -97,7 +96,6 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 			ExistenceReason::DepositHeld(_) => result = Keep,
```

Comment:
sidenote: thinking again I think here we should return `Remove` as we are forcing the removal, and caller is suppose to remove the account.
But anyway we only force in `do_destroy` and in this case we don't use the value returned

Reply:
As I first read this code I thought that it was a bug :laughing:  
I suppose you mean `result = Remove` as opposed to `return Remove`? Changed it to the former.

Reply:
yes you understand me right, I edited my message :-)